python/pyspark/ml/torch/distributor.py

```diff
@@ -85,6 +130,12 @@ def __init__(
         self.num_tasks = self._get_num_tasks()
         self.ssl_conf = None
 
+    def _create_input_params(self) -> dict[str, Any]:
```

Comment:
Why do we need this instead of passing a `kwargs`?

Reply:
This is just to pass the input params (specifically `num_processes` and `local_mode`) for the `staticmethod`s to use. We will have to have `kwargs` support in a future PR in case the users want to have a `max_restarts` option.